src/jit/codegencommon.cpp

```diff
@@ -10853,6 +10853,139 @@ unsigned CodeGen::getFirstArgWithStackSlot()
 
 #endif // !LEGACY_BACKEND && (_TARGET_XARCH_ || _TARGET_ARM64_)
 
+void CodeGen::genSinglePush()
```

Comment:
How about moving the definitions of genSinglePusy/Pop() to codegen.h file since they are trivially in-lineable?